CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_netsuite_source v0.5.1
+
+## Features 🎉 (affects Netsuite2 users only)
+- Introduces the `netsuite2__multibook_accounting` and `netsuite2__using_vendor_categories` variables to disable their related source tables and downstream models ([#21](https://github.com/fivetran/dbt_netsuite_source/pull/21)).
+  - `netsuite2__multibook_accounting` is `True` by default. Set it to `False` if you do not use the [Multi-Book Accounting](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/book_3831565332.html) feature in Netsuite and/or do not have the `accountingbook` and `accountingbooksubsidiaries` source tables.
+  - `netsuite2__using_vendor_categories` is `True` by default. Set it to `False` if you do not categorize vendors in Netsuite and/or do not have the `vendorcategory` source table.
+
```

Comment:
I'm not sure which place would be more appropriate to add it if you decide to, but how do you feel about adding a short blurb about finding these mappings in the records catalog? While I could see it clarifying questions..it may add more - but it also isn't easy to find the records catalog. So curious to hear what your thoughts are!